.github/workflows/stale.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '00 19 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3
+        with:
+          stale-issue-message: |
+            This issue is marked as stale because feedback has been requested for 30 days with no response. Please respond within 14 days or this issue will be closed due to inactivity.
+          close-issue-message: |
+            This issue was closed due to inactivity. If you can still reproduce this bug, please comment with the requested information, detailed steps to reproduce the problem, or any other notes that might help in the investigation.
+          start-date: '2021-06-01'
+          days-before-stale: 1
+          days-before-close: 1
+          stale-issue-label: stale
+          any-of-labels: 'bot-test'
+          exempt-milestones: 'Backlog,Discussion'
```

Comment:
Also exempt our other milestones? Just thinking a blanket no-milestones requirement makes sense.

Reply:
Good idea, commenting to remind myself when this opens back up.